.github/workflows/build-appimage.yml

```diff
@@ -0,0 +1,61 @@
+
+name: Build Gource AppImage
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+
+      - name: Install dependencies
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y \
+            build-essential cmake libsdl2-dev libpcre3-dev libglew-dev \
+            libglm-dev libboost-filesystem-dev libfreetype6-dev libpng-dev \
+            libjpeg-dev libfftw3-dev libvorbis-dev libopenal-dev libxrandr-dev \
+            libxi-dev libsdl2-* libfuse-dev
```

Comment:
Not sure it needs libfftw3-dev, libvorbis-dev, libopenal-dev, libxrandr-dev, libxi-dev or libfuse-dev.

Instead of libsdl2-* I think it just needs libsdl2-image-dev ?

Reply:
The libfuse was dependency was definitely needed when I was setting up the Ubuntu 24.04 container. I'll check the others and see. I think you're right about the libsdl2 dep too, I just did the * initially cause I wasn't sure which was needed. Let me make those changes and see if it'll build.